README.md

```diff
@@ -12,7 +12,7 @@ The `main` branch is the development branch: do not use this for production. Ref
 CDT currently supports Linux x86_64 Debian packages. Visit the [release page](https://github.com/AntelopeIO/cdt/releases) to download the package for the appropriate version of CDT. This is the fastest way to get started with the software.
 ### Debian package install
 
-The latest version of CDT is 3.0. Download the appropriate version of the Debian package and then install as follows:
+Download the appropriate version of the Debian package and then install. To download and install the latest version, run the following:
```

Comment:
and then install `it`. 
right? not a blocker :)

Reply:
Fixed